app/src/main/kotlin/com/intive/picover/validators/TextValidator.kt

```diff
@@ -0,0 +1,42 @@
+package com.intive.picover.validators
+
+import com.intive.picover.validators.ValidationStatus.BlankText
+import com.intive.picover.validators.ValidationStatus.EmptyText
+import com.intive.picover.validators.ValidationStatus.TooLongText
+import com.intive.picover.validators.ValidationStatus.ValidText
+
+class TextValidator private constructor(
+	private val text: String,
+	private val allowEmpty: Boolean?,
+	private val allowBlank: Boolean?,
+	private val maxLength: Int?,
+) {
+
+	fun validatingText(): ValidationStatus {
```

Comment:
minor: empty lines are not needed for the if's as well :)

Reply:
Done 😄